pyproject.toml

```diff
@@ -195,7 +195,7 @@ indent-style = "space"
 skip-magic-trailing-comma = false
 line-ending = "lf"
 
-[tool.ruff.mccabe]
+[tool.ruff.lint.mccabe]
```

Comment:
Ruff deprecated lint configurations in the top-level `ruff` namespace -> That's why I moved it into `ruff.lint`